docs/overview.rst

```diff
@@ -19,7 +19,7 @@ The use case is an |ak-tm| event streaming application that processes real-time
 
 The full event streaming platform based on |cp| is described as follows.
 Wikimedia's `EventStreams <https://wikitech.wikimedia.org/wiki/Event_Platform/EventStreams>`__ publishes a continuous stream of real-time edits happening to real wiki pages.
-A Kafka source connector `kafka-connect-sse <https://www.confluent.io/hub/cjmatta/kafka-connect-sse>`__ streams the server-sent events (SSE) from https://stream.wikimedia.org/v2/stream/recentchange, and a custom |kconnect| transform `kafka-connect-json-schema <https://www.confluent.io/hub/jcustenborder/kafka-connect-json-schema>`__ extracts the JSON from these messages and then are written to a |ak| cluster.
+A Kafka source connector `kafka-connect-sse <https://www.confluent.io/hub/cjmatta/kafka-connect-sse>`__ streams the server-sent events (SSE) from https://stream.wikimedia.org/v2/stream/recentchange, and a custom |kconnect| transform `kafka-connect-json-schema <https://www.confluent.io/hub/jcustenborder/kafka-connect-json-schema>`__ extracts the JSON from these messages and then the messages are written to a |ak| cluster.
```

Comment:
@joel-hamill For clarification, the extracted JSON is written; in contrast, the message w/out the JSON is NOT written.  So I think the PR is a little ambiguous on this.

Reply:
@ybyzek okay can you suggest what this should be?

Reply:
```...and then the content is written to a |ak| cluster.```?

Reply:
👍 SGTM